desktop/cmp/grid/impl/filter/ColumnHeaderFilter.ts

```diff
@@ -38,7 +38,6 @@ export const columnHeaderFilter = hoistCmp.factory({
             ),
             popoverClassName: 'xh-popup--framed',
             position: 'right-top',
```

Comment:
Do we need `rootBoundary: viewport` -- see link below

https://github.com/palantir/blueprint/wiki/Popover2-migration

Reply:
Good catch!